### PR TITLE
Update dns_srv_enum.rb

### PR DESCRIPTION
**Pull request by fraf0**
Patch for :
- Split record srvrcd one entry by line for readability.
- Add record for Default-First-Site-Name :
  (according to https://technet.microsoft.com/en-us/library/cc759550%28v=ws.10%29.aspx)
  - '_gc._tcp.Default-First-Site-Name._sites.',
  - '_kerberos._tcp.Default-First-Site-Name._sites.',
  - '_kerberos.tcp.Default-First-Site-Name._sites.dc._msdcs.',
  - '_ldap._tcp.Default-First-Site-Name._sites.',
  - '_ldap._tcp.Default-First-Site-Name._sites.dc._msdcs.',
  - '_ldap._tcp.Default-First-Site-Name._sites.gc._msdcs.',
- Remove double entry '_kerberos.tcp.dc._msdcs.'
- Add fqdn query in logs.
- Add report_note to store and preserve the fqdn query.

Ps : I'm not very familiar with the code and patch rules for modules. Thank you to excuse my eventual errors.

### CODE REVIEW

**Comment by wvu-r7** [comment]
Rules are in https://github.com/rapid7/metasploit-framework/blob/master/CONTRIBUTING.md.

File: modules/auxiliary/gather/dns_srv_enum.rb [hunk 1]
```diff
@@ -1,6 +1,21 @@
 ##
 # This module requires Metasploit: http://metasploit.com/download
 # Current source: https://github.com/rapid7/metasploit-framework
+#
```
Comment (wvu-r7) [inline modules/auxiliary/gather/dns_srv_enum.rb]: You can remove this comment block, since your patch is recorded in commits.

**Comment by fraf0** [comment]
Hello,
For wvu-r7 :
  Thanks for the link about rules.
  I made a new commit to remove the comment you indicate (this comment came from original code and not from my patch).

Regards,
fraf0

**Comment by wvu-r7** [comment]
@fraf0: I believe you removed the wrong comment. I was indicating the comment you added describing the patch, which is already recorded in your first commit.

**Comment by fraf0** [comment]
@wvu-r7  : Oups. Corrected.

**Comment by wvu-r7** [comment]
https://github.com/rapid7/metasploit-framework/commit/e98570cbd1ea1e239e1186e7efbff4c6770c83ba

**Pull request merged**